src/pages/index.js

```diff
@@ -285,6 +285,13 @@ profileEditButton.addEventListener("click", () => {
   ]);
   openModal(editModal);
 });
+
+editModal.addEventListener("keydown", (evt) => {
+  if (evt.key === "Escape") {
+    closeModal(editModal);
```

Comment:
Please, delete it

The checklist says: an event listener for closing the popup by pressing Esc is added when the popup is opened, and is removed when it's closed.